index.html

```diff
@@ -551,7 +551,40 @@ <h4>South Asian (Indic script) languages</h4>
 
         </section>
 
-         </section>
+		<section id="cjk-ivs">
+			<h4>East Asian languages</h4>
+
+			<p>The Chinese, Japanese, and Korean (CJK) writing systems sometimes encounter this issue.</p>
+
+			<p>A single ideograph can have multiple variants depending on the variation selector (VS) used. For example, <span class="codepoint" translate="no"><bdi lang="zh">&#x9F8D;</bdi><code class="uname">U+9F8D CJK UNIFIED IDEO­GRAPH-​9F8D</code></span> when followed by different variation selectors, such as <span class="codepoint" translate="no"><img alt="Variation Selector-17" src="images/E0100.png"><code class="uname">U+E0100 VARIATION SELECTOR-17</code></span> or <span class="codepoint" translate="no"><img alt="Variation Selector-18" src="images/E0101.png"><code class="uname">U+E0100 VARIATION SELECTOR-18</code></span>, may represent distinct glyphs. String searching operations need to consider whether matches are performed on the ideograph alone or on the full Ideographic Variation Sequence (<abbr>IVS</abbr>).</p>
```

Comment:
The language tag `zh` might not be specific enough to avoid local configuration issues in font selection. Considering using `zh-Hans` to ensure a simplified Chinese presentation?

---

Link term `ideograph` to the glossary. Consider creating IVS as a term in this document's Terminology section, with a pointer to the Unicode glossary entry [here](https://www.unicode.org/glossary/#ideographic_variation_sequence)

---

> may represent distinct glyphs

I think you need an illustration here. In fact, it might be clearer to describe the problem more directly. The introductory section of UTS#37 might be repeating or adaptation here:

> Characters in the Unicode Standard can be represented by a wide variety of glyphs. Occasionally the need arises in text processing to restrict or change the set of glyphs that are to be used to display a character. In special circumstances, this restriction needs to be expressed in plain text rather than by font selection or some other rich text mechanism. The Unicode Standard accommodates those circumstances with variation selectors: the code point of a graphic character can be followed by the code point of a variation selector to identify a restriction on the graphic character. The combination of a graphic character and a variation selector is known as a variation sequence (see Section 23.4, Variation Selectors of [[Unicode](https://www.unicode.org/reports/tr37/#Unicode)]).
>
> In the case of Han and other ideographs, it is impossible to build a single collection of variation sequences that can satisfy all the needs of the users. The requirements from scholars, governments and publishers are too different to be accommodated by a single collection. Instead they can be met by having multiple independent collections. Unicode defines an Ideographic Variation Database [[UTS#37] to ensure that there is a single definition for each given variation sequence, to make interchange of text using such variation sequences reliable.

Reply:
Thanks, Addison. Fixed.